src/presentations/indexable-presentation.php

```diff
@@ -319,6 +319,10 @@ public function generate_twitter_title() {
 			return $this->model->twitter_title;
 		}
 
+		if ( $this->model->og_title ) {
+			return $this->model->og_title;
+		}
+
 		if ( $this->title ) {
 			return $this->title;
 		}
```

Comment:
The `og_title` already includes this, so not strictly necessary. But putting it here makes the intention clear. Not sure, what do you think?

Reply:
@igorschoester Not sure I understand. This code says to fall back on `og_title` in case `twitter_title` cannot be found in the model. Firstly, I don't see this bit of code in `generate_og_title`, and secondly, it should be present in `generate_twitter_title` because otherwise it wouldn't be clear that this fallback should be used for `twitter_title`?

Reply:
I made an error here. You are right it should be there. I was thinking the `$this->model->og_title` would call the method `generate_og_title`, but it just grabs the indexable entry. So the fallback is not there yet.
So it is good the way it is 👍